### DISCUSSION TITLE
Kotlin Multiplatform

### DISCUSSION THREAD

**Discussion by Anthony17serrato** [android/nowinandroid#1665] [Ideas]
Hello, I am curious if this project has any plans to integrate with Kotlin Multiplatform? I think it would be really neat to see googles vision for a Modular KMP project.

**Comment by dturner** [comment]
Definitely interested in this, however, this is first and foremost a demonstration of Android best practices and we're currently struggling to keep up with those so unlikely that we'll broaden scope to other platforms any time soon.

**Comment by lihenggui** [comment]
Check this: https://github.com/android/nowinandroid/discussions/1415
